amplify/function/BusinessLogic/AssignUsersToTeams/resource.ts

```diff
@@ -0,0 +1,7 @@
+import { defineFunction } from "@aws-amplify/backend";
+
+export const AssignUsersToTeams = defineFunction({
+  entry: "./handler.ts",
+  runtime: 20,
+  name: "AssignUsersToTeams",
```

Comment:
I know this was originally empty for defaults, I just filled these out since it helps for code clarity when troubleshooting.

Reply:
This comment still needs to have resource information populated. It is also important that we set Node Version to 20 to ensure we are using a runtime that will be supported for a few years at least.